package.json

```
@@ -3,7 +3,8 @@
   "private": true,
   "version": "1.0.0",
   "scripts": {
-    "test": "pnpm run generate && jest",
+    "test": "pnpm run generate --skip-nx-cache && jest",
```

Is the skip cache still required when the task graph is properly wired up?

---

With the discovery of the third bullet, no, not really. Do you think we should revert it or keep it?

---

If we don't really need it then might be worth reverting. Not exactly a high effort step though, so not a huge deal.
```suggestion
    "test": "pnpm run generate && jest",
```